packages/backend/src/managers/playground.ts

```diff
@@ -56,6 +59,33 @@ export class PlayGroundManager {
     this.queries = new Map<number, QueryState>();
   }
 
+  async adoptRunningPlaygrounds() {
+    const containers = await containerEngine.listContainers();
```

Comment:
@benoitf this call returns an empty list when the extension is loaded at startup (and the correct list of containers when the extension is reloaded, after podman desktop has finished initialization).

What would be the right way to wait for the container engine to be up and running?

Reply:
Another solution would be to use `containerEngine.onEvent`, but it seems that we don't get any event when the containerEngine is started. @benoitf WDYT?

Reply:
We need to materialize the connection from AI studio to podman machine at a central point I'm pretty sure there is some code that try to get this connection

Reply:
I'm finally using `provider.onDidRegisterContainerConnection` to get informed.